python/kwiver/vital/algo/metadata_map_io.h

```diff
@@ -0,0 +1,26 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef KWIVER_VITAL_PYTHON_METADATA_MAP_IO_H_
+#define KWIVER_VITAL_PYTHON_METADATA_MAP_IO_H_
+
+#include <pybind11/pybind11.h>
+
+namespace kwiver {
+
+namespace vital {
+
+namespace python {
+
+namespace py = pybind11;
```

Comment:
In the algo binding header files, this using directive is typically declared outside of the `kwiver::vital::types` scope

Reply:
Thanks